pkgs/by-name/ad/adminer/package.nix

```diff
@@ -23,6 +23,11 @@ stdenv.mkDerivation (finalAttrs: {
     unzip
   ];
 
+  # Patch composer.json by inserting a "version" field using the derivation’s version.
+  patchPhase = ''
```

Comment:
Overriding `patchPhase` breaks `patches` field. Use `prePatch` or `postPatch` instead. See <https://github.com/jtojnar/nixpkgs-hammering/blob/master/explanations/patch-phase.md> for more info.

Also, the comment should be inside the string so that if more commands are added, it is not unclear which it refers to.

And comments should not just restate what the code is not doing, that is pretty clear. They should rather say why the code is there (e.g. gets rid of warning from composer run: text of the warning)

Reply:
Though in this case I would probably ignore the warning.